### ISSUE TITLE
[release] A JavaScript error occurred in the main process

### ISSUE THREAD

**Issue by veado** [thorchain/asgardex-electron#862]
By running latest release on Linux / Ubuntu.

### How to reproduce (A)?

1. Download `ASGARDEX-0.0.3-linux.AppImage` from [Draft release page](https://github.com/thorchain/asgardex-electron/releases/tag/untagged-2f3bba927f5adf0f91a4)

2. Make it executable 

    ```bash
    chmod +x {path-to}/ASGARDEX-0.0.3-linux.AppImage
    ```
3. Run it

    ```bash
    ./{path-to}/ASGARDEX-0.0.3-linux.AppImage
    ```
4. Output
    ```bash
    A JavaScript error occurred in the main process
    Uncaught Exception:
    TypeError: Cannot read property 'indexOf' of undefined
    at Function.t.getFileName (/tmp/.mount_ASGARDiiZg71/resources/app.asar/build/electron.js:18:62014)
    at t (/tmp/.mount_ASGARDiiZg71/resources/app.asar/build/electron.js:18:61164)
    at Object.<anonymous> (/tmp/.mount_ASGARDiiZg71/resources/app.asar/build/electron.js:22:45580)
    at r (/tmp/.mount_ASGARDiiZg71/resources/app.asar/build/electron.js:1:110)
    at Module.<anonymous> (/tmp/.mount_ASGARDiiZg71/resources/app.asar/build/electron.js:67:62068)
    at r (/tmp/.mount_ASGARDiiZg71/resources/app.asar/build/electron.js:1:110)
    at Object.<anonymous> (/tmp/.mount_ASGARDiiZg71/resources/app.asar/build/electron.js:67:134212)
    at r (/tmp/.mount_ASGARDiiZg71/resources/app.asar/build/electron.js:1:110)
    at Object.<anonymous> (/tmp/.mount_ASGARDiiZg71/resources/app.asar/build/electron.js:67:133460)
    at r (/tmp/.mount_ASGARDiiZg71/resources/app.asar/build/electron.js:1:110)

    ```
    
 ### How to reproduce (B)?
 
 1. Checkout latest `master`
 2. Package app locally by running `yarn package`
 3. Make `*.Appimage` executable 
 
    ```bash
    chmod +x {path-repo}/release/ASGARDEX-0.0.3-linux.AppImage
    ```
  3. Run it
    `./{path-to-repo}/release/ASGARDEX-0.0.3-linux.AppImage`
    
   4. Output
       Same error as above ^:
       ```bash
        A JavaScript error occurred in the main process
        Uncaught Exception:
        TypeError: Cannot read property 'indexOf' of undefined
        ...
      ```

**Comment by veado** [comment]
Temporary fixed by #863

**Comment by veado** [comment]
Finally fixed by #881
